### DISCUSSION TITLE
Home Assistant - Energie Dashboard - Zählerstand Wert im Ganzen hinzugefügt

### DISCUSSION THREAD

**Discussion by maglat** [jomjol/AI-on-the-edge-device#2828] [General]
Hallo zusammen,

ich habe aktuell folgendes Problem. Der ESP mit AONTED ließt meinen Wasserzähler sehr zuverläßig und pusht den Wert an mein Home Assistant via MQTT. In HA, habe ich den Value ins Energie Dashboard eingefügt. Dieser Verrechnet in der Regel, den letzten Stand mit dem aktuellen und Speichert die Differenz für die Stunde in der Statistik ab. Nun habe ich alle paar Tage aber das Problem, das HA einfach den Totalen Wasserverbauchswert dazu Addiert. Er verrechnet also nicht die Differenz des vorherigen. So bekomme ich in der Statistik auf komplett absurde Werte.  Manchmal nimmt er diesen sogar mal 2x oder 3x. Dann habe ich in einer Stunde 300t Liter Wasser verbraucht. Das tritt völlig Random auf. Mal läufts ein paar Tage sauber, manchmal am selben Tag einfach den Gesamtwasserverbauch nochmal on Top. Ich habe keine Ahnung was HA hier verhaut. Habt ihr das selber Problem auch schon gehabt?

**Comment by maglat** [comment]
Ich habe mal im Logbuch von HA geschaut, in einer der Zeit, als der Gesamtwert sogar 2x hinzu addiert worden ist. In dieser Zeitspanne hatten ich 2x "wechselt zu 17N.7119". Wisst ihr was das sein soll? Ich glaube, hier liegt der Hund begraben. Aber ich hab keine Ahnung was dieser komische Wert bedeutet.

![Bildschirmfoto 2024-01-26 um 22 56 57](https://github.com/jomjol/AI-on-the-edge-device/assets/44977256/a2125056-2290-4f82-8bfb-2b46cb7df6a3)

**Reply by sorbas** [reply]
Versuch mal nicht den "raw" Wert auszulesen, sondern arbeite mit dem "value" oder "pre" Wert. Diese sind zu einem höheren Grad von Artefakten bereinigt, die die KI nicht richtig erkannt hat.
